### ISSUE TITLE
On test login to http://localhost:3000/home - This page could not be found.

### ISSUE THREAD

**Issue by vistar** [omnivore-app/omnivore#4505]
With this manual:

> 3. Login with the test account
> During database setup docker compose creates an account demo@omnivore.app, password: demo_password.

And error in console:

```
omnivore-api               | [ioredis] Unhandled error event: Error: connect ENOENT %27redis://redis:6379%27
omnivore-api               |     at PipeConnectWrap.afterConnect [as oncomplete] (node:net:1494:16)
omnivore-api               |     at PipeConnectWrap.callbackTrampoline (node:internal/async_hooks:130:17)
```

**Comment by Podginator** [comment]
You have your redis in quotes. 

'redis://redis:6379'
remove the quotes.

**Comment by omerdn1** [comment]
@Podginator That fixes the redis error but the home page is still unavailable.

**Comment by omerdn1** [comment]
@Podginator Routes are available if you add  `/l/*` - e.g: localhost:3000/l/home
But then links are broken

**Comment by Podginator** [comment]
Yes, the config on the main branch writes an empty object to the next js file. If you’re trying to self host I’d look at the self hosting branch, where this has been changed slightly. Basically you need to ensure that the next.js config files are included. These automatically redirect /home to /l/home etc.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
